desktop/core/src/desktop/js/apps/storageBrowser/StorageFilePage/StorageFilePage.util.ts

```diff
@@ -25,37 +25,35 @@ export type MetaData = {
   value: string;
 };
 
-export const getFileMetaData = (t: TFunction, fileData: PathAndFileData): MetaData[][] => {
+export const getFileMetaData = (t: TFunction, fileStats: FileStats): MetaData[][] => {
   return [
     [
       {
         name: 'size',
         label: t('Size'),
-        value: formatBytes(fileData.stats?.size)
+        value: formatBytes(fileStats.size)
       },
       {
         name: 'user',
         label: t('Created By'),
-        value: fileData.stats?.user
+        value: fileStats.user
       }
     ],
     [
       {
         name: 'group',
         label: t('Group'),
-        value: fileData.stats?.group
+        value: fileStats.group
       },
       {
         name: 'permissions',
         label: t('Permissions'),
-        value: fileData.rwx
+        value: fileStats.rwx
       },
       {
         name: 'mtime',
         label: t('Last Modified'),
-        value: fileData.stats?.mtime
-          ? formatTimestamp(new Date(Number(fileData.stats.mtime) * 1000))
-          : '-'
+        value: fileStats.mtime ? formatTimestamp(new Date(Number(fileStats.mtime) * 1000)) : '-'
```

Comment:
If fileStats.mtime is a number, like interface states, then is Number() needed? 

Also, not for this PR but you should look into this `formatTimestamp` and improve it so that it uses a dynamic locale and can use a preset local_time like we discussed.

Reply:
Nice catch, thanks